org.opencpn.OpenCPN.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2018 Alec Leamas
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+
+# This manifest is used to build the official flathub package. 
+
+
+app-id: org.opencpn.OpenCPN
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
```

Comment:
Whatever version you end up choosing. The runtime version is a string, it needs a double quote to avoid weird yaml parsing.

Reply:
Right, long overdue. Fixed in  8559557